### PR TITLE
Update remote_webdriver_client.en.md

### PR DESCRIPTION
**Pull request by jt-pham**
I added some Remote WebDriver and DesiredCapabilities examples in Python

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Comment by Harsha509** [comment]
Hi @jt-pham,

Can you please update the sample code in all the translated pages too. 

Thanks!

File: docs_source_files/content/remote_webdriver/remote_webdriver_client.en.md [hunk 1]
```diff
@@ -18,7 +18,13 @@ driver.get("http://www.google.com");
 driver.quit();
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# We don't have a Python code sample yet -  Help us out and raise a PR  
+ from selenium import webdriver
+ from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
```
Comment (diemol) [inline docs_source_files/content/remote_webdriver/remote_webdriver_client.en.md]: Please do not use DesiredCapabilities, you can follow the other examples and use FirefoxOptions

File: docs_source_files/content/remote_webdriver/remote_webdriver_client.en.md [hunk 2]
```diff
@@ -63,7 +69,19 @@ driver.get("http://www.google.com");
 driver.quit();
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# We don't have a Python code sample yet -  Help us out and raise a PR  
+# https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities
+from selenium import webdriver
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
```
Comment (diemol) [inline docs_source_files/content/remote_webdriver/remote_webdriver_client.en.md]: Same comment as above.

**Comment by diemol** [comment]
For all watchers, this PR has gone stale, if anyone wants to submit it again, feel free to do so and please give credit to the original author.

**Comment by Harsha509** [comment]
> For all watchers, this PR has gone stale, if anyone wants to submit it again, feel free to do so and please give credit to the original author.

hi @jt-pham ,

Thanks for providing the PR.

 As the PR gone stale now, I've updated the sample code in PR https://github.com/SeleniumHQ/seleniumhq.github.io/pull/153

**Comment by diemol** [comment]
Replaced by #153

**Pull request closed without merging**